.github/workflows/_build_torch_xla.yml

```diff
@@ -37,6 +41,7 @@ jobs:
         with:
           repository: pytorch/pytorch
           path: pytorch
+          ref: ${{ inputs.torch-commit }}
```

Comment:
Can you make the same change to `_test.yml`? This is way cleaner than what I did:

https://github.com/pytorch/xla/blob/8471826a52417e010a21bbde70a28f426f7a2ad0/.github/workflows/_test.yml#L131-L144

Reply:
Do you mind if I do it in a follow up PR?

Reply:
Yeah, works for me.